scripts/build.sh

```diff
@@ -97,9 +97,22 @@ rm -rf ./proto/.git
 # Primary use is TLS between Broker-CLI and Broker Daemon
 #
 #############################################
-KEY_PATH="./certs/broker-rpc-tls.key"
-CERT_PATH="./certs/broker-rpc-tls.cert"
-CSR_PATH="./certs/broker-rpc-csr.csr"
+
+DIRECTORY=~/.sparkswap
```

Comment:
it feels like this should be configurable, perhaps via an environment variable or command-line argument.

Reply:
Just to add documentation to this. Since we do not allow the `~/.sparkswap` directory to be configurable for the CLI, we've instead added the task `Make ~/.sparkswap application directory configurable` to an upcoming roadmap